app/build.gradle.kts

```diff
@@ -68,6 +68,10 @@ dependencies {
     implementation("androidx.work:work-rxjava2:2.4.0")
     implementation("androidx.startup:startup-runtime:1.0.0")
 
+    // ML Kit barcode Scanner
+    implementation ("com.google.mlkit:barcode-scanning:16.1.1")
+
+    implementation ("android.arch.lifecycle:extensions:1.1.1")
```

Comment:
I think this is deprecated, and we should use androidx

Reply:
Yes. Thanks.